### PR TITLE
WIP: Feature/token sale

### PR DESCRIPTION
**Pull request by FredrikOseberg**
**What current issue(s) from Trello/Github does this address?**

**What problem does this PR solve?**

**How did you solve this problem?**

**How did you make sure your solution works?**

**Are there any special changes in the code that we should be aware of?**

**Is there anything else we should know?**

- [ ] Unit tests written?

### CODE REVIEW

File: app/components/TokenSale/TokenSaleHeader/index.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  <section className={styles.tokenSaleHeader}>
+    <h1>Token Sale</h1>
+    <button type="button">
+      <RefreshIcon />
+      Refresh
+    </button>
+  </section>
+)
+
+export default TokenSaleHeader
```
Comment (comountainclimber) [inline app/components/TokenSale/TokenSaleHeader/index.jsx]: Theres actually a common component (`<HeaderBar/>`) for this part of the UI (it gets rendered throughout the app) please use it and remove the files associated with TokenSaleHeader

File: app/components/Inputs/CheckBox/index.jsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
+// flow
```
Comment (comountainclimber) [inline app/components/Inputs/CheckBox/index.jsx]: For consistency with the rest of the application (I ignored this in your contribution with the Send work but would like to address it in the future) can we call this file CheckBox.jsx with another file called index.js that exports the component as well as everywhere else in this PR where this pattern exists

**Comment by comountainclimber** [comment]
Something funky definitely still going on with this diff... Theres no reason why all of these commits should also be going into design-v2 with your work here but we will get it sorted out. I will try and investigate whats going with your branch later today

Unrelated to the git stuff PR looks like a solid start nice work! One thing: Can we incorporate the ability to add network fees (not in the design mocks) to the "sale" flow - You can reference the work I did in the Send UX but its really as simple as adding a key called fees to the config here: https://github.com/CityOfZion/neon-wallet/blob/dev/app/modules/sale.js#L91

File: app/components/TokenSale/TokenSaleHeader/index.jsx (hunk 1 above)
Comment (FredrikOseberg) [reply app/components/TokenSale/TokenSaleHeader/index.jsx]: Perfect. Will implement.

File: app/components/Inputs/CheckBox/index.jsx (hunk 2 above)
Comment (FredrikOseberg) [reply app/components/Inputs/CheckBox/index.jsx]: No problem.

**Comment by FredrikOseberg** [comment]
Yeah something is definitely going on. Thanks for the review!

**Comment by comountainclimber** [comment]
Closed in favor of https://github.com/CityOfZion/neon-wallet/pull/1424

**Pull request closed without merging**